src/components/index-page/IndexMoreMomentum.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+
+import decorativeSquare from "../../assets/images/decorative_square.png";
+import "./css/index-more-momentum.css";
+
+export const IndexMoreMomentum = () => (
```

Comment:
let's remove the index prefix from the component name, css file name, and css classes. We don't want to assume that this section will always exist on the index page.